### ISSUE TITLE
Sarameya resetting Enmity instead of Hate

### ISSUE THREAD

**Issue by Mozzy4Ever** [EdenServer/community#5495]
### Checklist

- [X] I have searched for existing issues for issues like this one. The issue has not been posted.
- [X] I have provided reproducible steps. (No "as the title says" posts please. ***Please try to state facts only, and avoid assumptions/opinions.***)
- [X] I have provided my client version in the details. (type /ver into your game window)
- [X] ***I have provided proof of my claim (era videos, links to era wiki and/or forum posts)*** Lack of proof will slow down fixes and prioritization. Please provide details so that developers' research time is minimized.

### Details
/ver 3018120_0

According to:

https://ffxiclopedia.fandom.com/wiki/Sarameya?oldid=991639
https://wikiwiki.jp/ffxi/NM/%E8%8A%B1%E9%B3%A5%E9%A2%A8%E6%9C%88/%E3%83%88%E3%83%AD%E3%83%BC%E3%83%AB%E5%82%AD%E5%85%B5%E5%9B%A3#a7a1b3f5

and several livejournals, Cerberus is not an enmity list reset but a hate list reset. Currently on Eden, it's a enmity list reset. 
![image](https://user-images.githubusercontent.com/11255886/220544838-8b3195fb-9967-4728-8177-1be5fe0d2714.png)

**Comment by Steam-arch** [comment]
For what its worth, I tried checking this awhile ago threough era vids (I thought the same because of kanicans livejournal) but leaning towards its correct on eden because vids didnt really support the other theory. Aether also didnt find any proof on era test (pretty sure).

tldr: gonna need proof
